packages/ember-runtime/lib/system/core_object.js

```diff
@@ -100,9 +100,7 @@ function makeCtor() {
           var keyName = keyNames[j];
           var value = properties[keyName];
 
-          if (IS_BINDING.test(keyName)) {
-            m.writeBindings(keyName, value);
```

Comment:
is there a reason we a testing if bindings exist now, but no longer writing them? I suspect this deletion was in error, or I am mis understanding something.